bin/install.sh

```diff
@@ -4,31 +4,21 @@ set -e
 export BUILD_HARNESS_ORG=${1:-mintel}
 export BUILD_HARNESS_PROJECT=${2:-build-harness}
 export BUILD_HARNESS_BRANCH=${3:-master}
-export BUILD_HARNESS_TAG=${4:-latest}
 export GITHUB_REPO="https://github.com/${BUILD_HARNESS_ORG}/${BUILD_HARNESS_PROJECT}.git"
 
-echo "Installing ${BUILD_HARNESS_PROJECT}..."
-
-if [ "$BUILD_HARNESS_PROJECT" ] && [ -d "$BUILD_HARNESS_PROJECT" ]; then
-  echo "Updating from ${GITHUB_REPO}#${BUILD_HARNESS_BRANCH}"
-  (
-    cd $BUILD_HARNESS_PROJECT && \
-    git fetch origin --tags --force && \
-    git checkout $BUILD_HARNESS_BRANCH >/dev/null && \
-    git pull \
-  )
-else
-  echo "Cloning from ${GITHUB_REPO}#${BUILD_HARNESS_BRANCH}"
-  git clone -b $BUILD_HARNESS_BRANCH $GITHUB_REPO >/dev/null 2>&1
+if [ -z "$BUILD_HARNESS_BRANCH" ] || [ "$BUILD_HARNESS_BRANCH" == "master" ] ; then
```

Comment:
How would we clone the latest from master when developing the build-harness then?

Reply:
I was thinking we could just always test from a branch. Goals:

1. User can pin to a release/tag
2. User can pin to a tag older than the currently checked out version without running `clean`
2. User can pin to a branch (other than master)

I think this covers all the necessary use cases?

Reply:
(You can also still test master HEAD locally by just checking out master into your current path.)

Reply:
Good point.